docs/changelog.rst

```diff
@@ -33,6 +33,7 @@ Major new features:
 Other new things:
 
 * Permissions plugin now sets cover art permissions to the file permissions.
+* Unimported plugin now supports excluding specific subdirectories in library.
```

Comment:
I know the previous bullet doesn't do it, but the way we usually refer to changes in plugins is like this:

```rest
* :doc:`/plugins/unimported`: Add a new configuration option to exclude subdirectories in the library.
```

Reply:
Hi @sampsyo thanks for the quick review. How's the new code?